eventtracking/processors/filters.py

```diff
@@ -0,0 +1,44 @@
+"""
+Filter configured through admin panel. Filters out the events that
+do not need to be routed
+"""
+import logging
+
+from eventtracking.processors.exceptions import EventEmissionExit
+
+logger = logging.getLogger(__name__)
+
+
+class RegExFilterProcessor:
+    """
+    Filter the events using regex. The regex list Can be configured
+    through the Admin panel.
+    """
+
+    def __call__(self, event, **kwargs):
+        # To avoid the "Models aren't loaded yet" error,
+        # importing the model here
+        # pylint: disable=import-outside-toplevel
+        from eventtracking.django.models import RegExFilter
+        try:
+            regex_filter = RegExFilter.objects.all()[0]
+        except IndexError:
+            logger.warning('Regular Expression Filter does not have any configurations. '
+                           'Cannot filter events.')
+            return event
+
+        event_name = event.get('name')
+
+        if not regex_filter.is_enabled:
```

Comment:
`get_latest_filter` should return `None` if there is no enabled filter then we can do something like this
```
if not regex_filter
logger.info(
                'Regular Expression Filter is not enabled. Allowing the event %s to pass through.',
                event_name
            )
```